src/SwedbankPay/Api/Client/Version.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace SwedbankPay\Api\Client;
+
+/**
+ * Class Version
+ * @package SwedbankPay\Api\Client
+ */
+class Version
+{
+    /**
+     * Gets the version number from a defined constant VERSION, the environment
+     * variable VERSION or from composer.json.
+     *
+     * @return string Version number
+     * @throws ClientException
+     */
+    public static function getVersion()
+    {
+        $version = false;
+
+        if (defined('VERSION')) {
+            $version = VERSION;
+        } elseif (getenv("VERSION") !== false) {
+            $version = getenv("VERSION");
+        } else {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method getVersion uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-php/pullRequest?prid=5251412)